React/Views/ScrollView/RCTScrollView.m

```diff
@@ -106,6 +107,11 @@ + (BOOL)isCompatibleWithResponsiveScrolling
   return YES;
 }
 
+- (BOOL)isFlipped
+{
+  return !self.inverted;
```

Comment:
This is a fun one. Internally in Microsoft, we avoid dot notation in objective c as much as possible (see [aka.ms/obj-c-guide](https://microsoft.github.io/objc-guide/Syntax/DotNotation.html)). However, React Native Core doesn't have that as their style. For React Native macOS, I'm inclined to allow this to keep the macOS code similar to the iOS code and not flip styles inside our diffed files, so long as we don't have redundant method invocation. @harrieshin thoughts?

Reply:
i think generally any new code we write we try to avoid dot as much as we can. if we inherit from rncore yeah leave it as is